scene4c.rpy

```diff
@@ -0,0 +1,163 @@
+label scene4c;
+
+   #stage: JOY and enter from the right
+
+   JOY "Good thing I turned SID down, right?"
+
+   #stage: JOY turns to WOE
+
+   JOY "I'm not really into doing that, right? Or coworkers, right?"
+
+   menu: 
+      WOE 
+
+      "Or clients, for that matter":
+   $ choice = choices['scene4'] = 'sid'
+   JOY "FAITH was charming, but I do like keeping work and play seperate. sprite_JOY_happy"
+   jump scene4c_common
+
+   "The right person will come along1":
+   $ choice = choices['scene4'] = 'sid'
+   JOY "We've been telling ourselves that for years, Joy... sprite_JOY_sad"
+   jump scene4c_common
+
+   "We all die alone, you see":
+   $ choice = choices['scene4'] = 'sid'
+   JOY "Okay, I am really in some kind of state! Bad intrusive thoughts! sprite_JOY_angry"
+   jump scene4c_common
+
+label scene4c_common:
+
+   show sprite_JOY_neutral with moveleft
+
+   ??? "Hey uh... is this where i'm supposed to go for pets?"
+
+   Joy "Here we go..."
+
+   show sprite_DOPEY_neutral with moveinright
+
+   ??? "Uh, hello? I think my dog is like, and not okay and stuff?"
+
+   JOY "Yes, come on in. What's your name sir?"
+
+   DOPEY "Oh Uh, my name is DOPEY. What's up?"
+
+   JOY "Whats...up? Wahat's up is, you said your dog isn't feeling well?"
+
+   DOPEY "I mean I'm not super sure. It's really hard to tell."
+
+   JOY "What do you mean its hard to tell--"
+
+   show sprite_JOY_fear
```

Comment:
have a look at `game\assets\story\scene5a.rpy` by @trcrzs which has a lot of animation polish and makes use of hiding the prior sprites when emotions change